Grid.js

```diff
@@ -0,0 +1,168 @@
+import Cell from "./Cell.js";
+
+class Grid {
+    constructor(
+        gridSize,
+        numMines,
+    ) 
+    {
+        this.gridSize = gridSize;
+        this.numMines = numMines;
+        this.cells = [];
+    }
+
+    createGridCells() {
+        this.cells = new Array(this.gridSize);
+    
+        for (let i = 0; i < this.gridSize; i++) {
+            this.cells[i] = new Array(this.gridSize);
+        };
+    
+        for (let row = 0; row < this.gridSize; row++) {
+            for (let col = 0; col < this.gridSize; col++) {
+                let cell = new Cell(
+                    row,
+                    col,
+                    false,
+                    undefined,
+                    false,
+                    false
+                );
+    
+                this.cells[row][col] = cell;
+            };
+            
+        };
+    };
+
+    getCell(row, col) {
+        return this.cells[row][col];
+    }
+
+    changeCellProperty(row, col, prop, value){
+        this.cells[row][col][prop] = value;
+    }
+    
+    assignMines(){  
+        function getRandomInt(max) {
+            return Math.floor(Math.random() * max);
+        }
+        for (let i = 0; i < this.numMines;) {
+            let row = getRandomInt(this.gridSize);
+            let col = getRandomInt(this.gridSize);
+            let cell = this.getCell(row, col);
+            if (!cell.isMine) {
+                cell.isMine = true;
+                i++;
+            }
```

Comment:
I think this makes more sense conceptually as a `while` loop. `for` loops are usually used to iterate over a range (i.e. 1 to 10) while `while` loops are used to keep looping until a certain condition is met. Of course they can be used interchangeably, it's more a matter of best fit.
```suggestion
        let numMinesCreated = 0;
        while (numMinesCreated < this.numMines) {
            let row = getRandomInt(this.gridSize);
            let col = getRandomInt(this.gridSize);
            let cell = this.getCell(row, col);
            if (!cell.isMine) {
                cell.isMine = true;
                numMinesCreated++;
            }
```

Reply:
change of style/readability - talked about